modules/Emails/Email.php

```diff
@@ -3028,58 +3028,60 @@ public function send(
 
         ///////////////////////////////////////////////////////////////////////
         ////	ATTACHMENTS
-        if (isset($this->saved_attachments)) {
-            foreach ($this->saved_attachments as $note) {
-                $mime_type = 'text/plain';
-                if ($note->object_name == 'Note') {
-                    if (!empty($note->file->temp_file_location) && is_file($note->file->temp_file_location)) { // brandy-new file upload/attachment
-                        $file_location = "file://" . $note->file->temp_file_location;
-                        $filename = $note->file->original_file_name;
-                        $mime_type = $note->file->mime_type;
-                    } else { // attachment coming from template/forward
-                        $file_location = "upload://{$note->id}";
-                        // cn: bug 9723 - documents from EmailTemplates sent with Doc Name, not file name.
-                        $filename = !empty($note->filename) ? $note->filename : $note->name;
-                        $mime_type = $note->file_mime_type;
-                    }
-                } elseif ($note->object_name == 'DocumentRevision') { // from Documents
-                    $filePathName = $note->id;
-                    // cn: bug 9723 - Emails with documents send GUID instead of Doc name
-                    $filename = $note->getDocumentRevisionNameForDisplay();
-                    $file_location = "upload://$note->id";
-                    $mime_type = $note->file_mime_type;
-                }
-
-                // strip out the "Email attachment label if exists
-                $filename = str_replace($mod_strings['LBL_EMAIL_ATTACHMENT'] . ': ', '', $filename);
-                $file_ext = pathinfo($filename, PATHINFO_EXTENSION);
-                //is attachment in our list of bad files extensions?  If so, append .txt to file location
-                //check to see if this is a file with extension located in "badext"
-                foreach ($sugar_config['upload_badext'] as $badExt) {
-                    if (strtolower($file_ext) == strtolower($badExt)) {
-                        //if found, then append with .txt to filename and break out of lookup
-                        //this will make sure that the file goes out with right extension, but is stored
-                        //as a text in db.
-                        $file_location = $file_location . ".txt";
-                        break; // no need to look for more
-                    }
-                }
-                $mail->AddAttachment(
-                    $file_location,
-                    $locale->translateCharsetMIME(trim($filename), 'UTF-8', $OBCharset),
-                    'base64',
-                    $mime_type
-                );
+        if($_REQUEST['refer_action'] != 'ReplyTo' && $_REQUEST['refer_action'] != 'ReplyToAll'){
```

Comment:
There should be a space after the ```if``` and ```{``` here. The rest of this function also needs the spacing adjusted for PSR-2. Nice work!

Reply:
@Dillon-Brown Shall I create new PR after resolving spacing issue?